first/expo/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Farmington Valley FIRST Expo
+---
+
+<!--<h2>Farmington Valley FLL Jr.</h2>
+This section will be updated with more information about the Farmington Valley FLL Jr. programs soon.
+-->
+<h2>Farmington Valley <i>FIRST</i> Expo</h2>
+<img class="float-left" alt="Boomtown Build Logo" style="width:300px;" src="{{ site.url }}/assets/img/FIRST-logos/Boomtown-Build-logo.jpg"/>
+We run & host the Farmington Valley FIRST Expo (FVFE). Students plan every aspect of this event, from creating city-themed STEAM activities to coordinating logistics. Over the last 8 years, this unique event has grown to encompass every level of FIRST, including an FLL Jr. Expo, FLL scrimmage, & FTC/FRC robot demonstrations. There were 34 FLL Jr, 14 FLL, 1 FTC, & 5 FRC teams at this event to share their accomplishments & be inspired through experiential learning. The FVFE is a unique showcase of FIRST due to its focus on our youngest participants, guiding them to see what is possible at all levels of FIRST. On a Sunday during build season, we reached over 1000 people in this celebration of FIRST.
+<br /><br />
+
+
+## Activities from 2020
+Here are some actitvites that can be used for your own Expo events and FLL Jr. teams:
+* [Architect Assembly Structured Activity](https://drive.google.com/file/d/1gIQXTvs16d_LLYd8Gc_4ASJzXgzWFL2n/view){:target="_blank"}
+* [Delivery Truck Dash Structured Activity](https://drive.google.com/file/d/1KvIu9n1Sk6wrI985ahchIznALPnHjj1z/view){:target="_blank"}
+* [Gingerbread House Paper Circuits](https://drive.google.com/file/d/15ObwQ5PltB43tfF0R2fnmPSGbTPZCBqF/view){:target="_blank"}
+* [Pinwheel](https://drive.google.com/file/d/14vT7fuzKDc57NsDLsy2Ktseh-kDqmll7/view){:target="_blank"}
+* [Popsicle Stick Bridges Structured Activity](https://drive.google.com/file/d/1M6Sl4v65oLXQaTpIUOWftB4EywbpPA9X/view){:target="_blank"}
+* [Snowflake Activity](https://drive.google.com/file/d/1qSpENS9ygt8nuTwIUQ1PoEblq5DYB3mL/view){:target="_blank"}
```

Comment:
Similar to the pinwheel, I would match this one to the title. It's not called an activity on its PDF instructions.